packages/build/src/log/messages/core_steps.js

```diff
@@ -128,14 +128,19 @@ export const logSecretsScanSuccessMessage = function (logs, msg) {
   log(logs, msg, { color: THEME.highlightWords })
 }
 
-export const logSecretsScanFailBuildMessage = function ({ logs, scanResults, groupedResults }) {
+export const logSecretsScanFailBuildMessage = function ({
+  logs,
+  scanResults,
+  groupedResults,
+  enhancedScanShouldImpactBuilds,
```

Comment:
could this be me renamed as well to `enhancedScanShouldRunInActiveMode` to avoid using different names?

I do like the new names and approach, much less dizzy